src/main/java/org/apache/mesos/offer/OfferAccepter.java

```diff
@@ -45,7 +46,7 @@ public OfferAccepter(Set<OperationRecorder> recorders) {
   }
 
   public List<OfferID> accept(SchedulerDriver driver, List<OfferRecommendation> recommendations, Filters filters) {
-    if (recommendations.size() <= 0) {
+    if (CollectionUtils.isEmpty(recommendations)) {
```

Comment:
CollectionUtils.isEmpty includes a `null` check. If you feel that is not needed here, happy to switch to recommendations.empty().